Emergence/Contexts/Networking/ArtsyAPI.swift

```diff
@@ -79,9 +81,16 @@ extension ArtsyAPI : MoyaTarget {
         case .ArtworksForShow(let partnerID, let showID):
             return "/api/v1/partner/\(partnerID)/show/\(showID)/artworks/"
 
+        case .ImagesForShow(let showID):
+            return "/api/v1/show/\(showID)/images"
+
         case .RunningShowsNearLocation, .UpcomingShowsNearLocation, .PastShowsNearLocation:
             return "/api/v1/shows"
+
+        case .FeaturedShows:
+            return "/api/v1/set/530ebe92139b21efd6000071/items"
```

Comment:
Is this just for now, or one of those special links like in Eigen? Could use a comment.